src/bulk/common.ts

```diff
@@ -167,59 +163,48 @@ export class Batch<T = Document> {
   }
 }
 
+function generateIdMap(ids: Document[]): { [key: number]: any } {
```

Comment:
just a thought - this is only used on the bulk write result class.  for organization purposes, we could make it a private static method on the class.  

thoughts?

Reply:
Changed per suggestion.